ui-kit/src/main/kotlin/org/p2p/uikit/components/edittext/UiKitEditTextStyleAttrsApplier.kt

```diff
@@ -0,0 +1,82 @@
+package org.p2p.uikit.components.edittext
+
+import androidx.core.content.res.getColorOrThrow
+import androidx.core.content.res.getDrawableOrThrow
+import androidx.core.content.res.getIntOrThrow
+import androidx.core.content.res.getResourceIdOrThrow
+import androidx.core.content.res.getStringOrThrow
+import android.content.res.ColorStateList
+import android.content.res.TypedArray
+import android.graphics.drawable.Drawable
+import android.text.method.DigitsKeyListener
+import org.p2p.uikit.R
+import org.p2p.uikit.databinding.WidgetUiKitEdittextBinding
+import org.p2p.uikit.utils.withImageOrGone
+import org.p2p.uikit.utils.withTextOrGone
+
+internal class UiKitEditTextStyleAttrsApplier {
+    private companion object {
```

Comment:
all the available attrs that are REALLY working can be found here

i removed all the attributes connected to android: namespace to be more explicit.